docs/advisors/checks/postgresql-sequential-scan-check.md

```diff
@@ -0,0 +1,20 @@
+# PostgreSQL excessive sequential scans 
+
+## Description
+This checks for relations with excessive sequential scan vs index scans. Although the query planner will occasionally choose sequential scans  when it is more efficient than using an index ( typically low count for tuples ), this check is based on at least 50,000 live tuples in the relation.
+
+
+## Resolution
+Some basic steps can be taken to help with this problem. 
+- First of all, make sure the relations are analyzed regularly.  
+- Check pg_stat_user_tables for statistics about vacuums and analyze. 
+- Identify the queries using the relations noted in the check and run explain on them.  This will help identify if a relation is missing an index. 
```

Comment:
```suggestion
- Identify the queries using the relations noted in the check, and run EXPLAIN on them. This will help identify relations with missing indexes. 
```